jazz-ui/src/app/app.module.ts

```diff
@@ -1,198 +1,69 @@
-import { EnvDeploymentsSectionComponent } from './pages/environment-detail/env-deployments-section.component';
-import { EnvCodequalitySectionComponent } from './pages/environment-detail/env-codequality-section.component';
-import { EnvLogsSectionComponent } from './pages/environment-detail/env-logs-section.component';
-import { EnvOverviewSectionComponent } from './pages/environment-detail/env-overview-section.component';
-import { ServiceCostComponent } from './pages/service-cost/service-cost.component';
-import { BarGraphComponent } from './secondary-components/bar-graph/bar-graph.component';
-import { AmountComponent } from './primary-components/amount/amount.component';
-import { FiltersComponent } from './secondary-components/filters/filters.component';
-import { TableTemplateComponent } from './secondary-components/table-template/table-template.component';
-import { SearchBoxComponent } from './primary-components/search-box/search-box.component';
-
-import { DaterangePickerComponent } from './primary-components/daterange-picker/daterange-picker.component';
-import { MobileSecondaryTabComponent } from './secondary-components/mobile-secondary-tab/mobile-secondary-tab.component';
-import { JazzMobHeaderComponent } from './secondary-components/jazz-mob-header/jazz-mob-header.component';
-
-import { LineGraphComponent } from './secondary-components/line-graph/line-graph.component';
-import { ServiceMetricsComponent } from './pages/service-metrics/service-metrics.component';
-
-import { JazzToasterComponent } from './secondary-components/jazz-toaster/jazz-toaster.component';
-import { JenkinsStatusComponent } from './pages/jenkins-status/jenkins-status.component';
-import { FooterComponent } from './secondary-components/footer/footer.component';
-import { Error404Component } from './pages/error404/error404.component';
-import { RegisteredComponent } from './secondary-components/registered/registered.component';
-
-
-const appRoutes: Routes = [
-  {
-    path : '',
-    component : LandingComponent
-  },
-  {
-    path: 'landing',
-    component: LandingComponent
-  },
-  {
-    path: '404',
-    component: Error404Component
-  },
-  {
-    path: 'registered',
-    component: RegisteredComponent
-  },
-  {
-    path: 'approval',
-    component: JenkinsStatusComponent
-  },
-  {
-    path: 'services',
-    component: ServicesComponent,
-    children: [
-      {
-        path: '',
-        component: ServicesListComponent,
-        canActivate: [RouteGuard]
-      },
-      {
-        path: ':id',
-        component: ServiceDetailComponent,
-        canActivate: [RouteGuard]
-      },
-      {
-        path: ':id/:env',
-        component: EnvironmentDetailComponent,
-        canActivate: [RouteGuard]
-      }
-    ]
-  },
-  { path: ':',
-    redirectTo: '404',
-    pathMatch: 'full'
-  }
-];
+let environmentval = environment.envName;
+let loadedModule;
+if(environmentval == "oss"){
```

Comment:
indentation here and other places in the code

Reply:
did the changes